libqtile/scripts/migrations/rename_unspecified.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2023, elParaguayo. All rights reserved.
```

Comment:
Minor point but this is yours and this year...

Other than that, this all looks good to me.

Reply:
urgh, yep. I wonder if we can switch to SPDX headers vs. having this giant blurb everywhere. MIT makes it hard i guess.

Reply:
Oh, it does have an SPDX header: https://spdx.org/licenses/MIT.html

It's just debian that won't define MIT. So maybe we can switch and get rid of all of this crap!